pynest/nest/tests/test_stdp_triplet_synapse.py

```diff
@@ -84,11 +85,11 @@ def decay(self, time, Kplus, Kplus_triplet, Kminus, Kminus_triplet):
 
     def facilitate(self, w, Kplus, Kminus_triplet):
         """Facilitate weight."""
-        return w + Kplus * (self.syn_spec["Aplus"] + self.syn_spec["Aplus_triplet"] * Kminus_triplet)
+        return abs(w + Kplus * (self.syn_spec["Aplus"] + self.syn_spec["Aplus_triplet"] * Kminus_triplet))
```

Comment:
After incorporating the changes proposed by @heplesser in https://github.com/nest/nest-simulator/pull/284#discussion_r58529377 the test functions for ``facilitate`` and ``depress``, and calls to them, should be adapted accordingly.